src/main/com/homework/week9/invoice/service/InvoiceService.java

```diff
@@ -0,0 +1,61 @@
+package com.homework.week9.invoice.service;
+
+
+public class InvoiceService {
+    public Invoice[] load() {
+        InvoiceReader invoiceReader = new InvoiceReader();
+        return invoiceReader.read();
+    }
+
+    public long totalAmountByType(Enum type) {
+        long sum = 0;
+        Invoice[] invoices = load();
```

Comment:
please keep Invoice[] array as a global variable and assign its value in load() method